ai4good/models/cm/initialise_parameters.py

```diff
@@ -217,21 +217,27 @@ def __init__(self, ps: ParamStore, camp: str, profile: pd.DataFrame, profile_ove
         self.generated_disease_vectors = self.ps.get_generated_disease_param_vectors()
 
     def sha1_hash(self) -> str:
-        hash_params = [
-            {i: self.control_dict[i] for i in self.control_dict if i != 'nProcesses'},
-            self.infection_matrix.tolist(),
-            self.im_beta_list.tolist(),
-            self.largest_eigenvalue,
-            self.generated_disease_vectors.to_dict('records'),
-            self.population_frame.to_dict('records'),
-            self.population,
-            self.camp,
-            self.calculated_categories,
-            self.model_params.to_dict('records')
-        ]
-        serialized_params = json.dumps(hash_params, sort_keys=True)
-        hash_object = hashlib.sha1(serialized_params.encode('UTF-8'))
-        _hash = hash_object.hexdigest()
+        # hash_params = [
+        #     {i: self.control_dict[i] for i in self.control_dict if i != 'nProcesses'},
+        #     self.infection_matrix.tolist(),
+        #     self.im_beta_list.tolist(),
+        #     self.largest_eigenvalue,
+        #     self.generated_disease_vectors.to_dict('records'),
+        #     self.population_frame.to_dict('records'),
+        #     self.population,
+        #     self.camp,
+        #     self.calculated_categories,
+        #     self.model_params.to_dict('records')
+        # ]
+        # serialized_params = json.dumps(hash_params, sort_keys=True)
+        # hash_object = hashlib.sha1(serialized_params.encode('UTF-8'))
+        # _hash = hash_object.hexdigest()
+
+        #change hash temporarily to change the resid output for better readability
+        control_params = {i: self.control_dict[i] for i in self.control_dict if i != 'nProcesses' and i != 't_sim' and i != 'numberOfIterations'}
+        print(control_params)
+        params_string=f"hyg_{control_params['better_hygiene']['value']}_{control_params['better_hygiene']['timing']}" + "_" + f"ICU_{control_params['ICU_capacity']['value']:.5f}" + "_" + f"rmSym_{control_params['remove_symptomatic']['timing']}_{control_params['remove_symptomatic']['rate']:.5f}" + "_" + f"shield_{control_params['shielding']['used']}" + "_" + f"rmRisky{control_params['remove_high_risk']['timing']}_{control_params['remove_high_risk']['rate']:.5f}"
+        _hash=self.camp+'_'+params_string
```

Comment:
since in practice we likely change various params, like camp or beta list I believe we need hash. I suggest we write small command line tool that lists all cached values and pretty prints important parameters instead of this